pkgs/tools/typesetting/tex/texlive/default.nix

```diff
@@ -10,6 +10,9 @@
 , libfaketime, asymptote, biber-ms, makeFontsConf
 , useFixedHashes ? true
 , recurseIntoAttrs
+, extraName ? "combined-infraonly"
+, extraOutputsToInstall ? [ "man" ]
+, requiredTeXPackages ? (ps: [ ps.scheme-infraonly ])
```

Comment:
I have opted for using what we already have:
```nix
texlive.override { extraOutputsToInstall = [ "man" "texdoc" ]; }
texlive.withPackages (ps: [ ... ])
```
instead of exposing a new `buildEnv`, which would be something new to maintain. I think that this is where I would prefer to go.

The PR is still incomplete in that `texlive.combined.scheme-small.override` has the arguments of `__buildEnv` rather than coming from `callPackage`, and that would be weird in practice.

Reply:
Changed my mind drastically.